src/Routes.js

```diff
@@ -90,6 +91,7 @@ const Routes = (props) => (
     <AuthenticatedRoute path="/admin_manage_lc/" component={AdminManageLC} props={props} rolesAllowed={[USER_ROLES.admin]} />
     <AuthenticatedRoute path="/admin_manage_dar_collections/" component={AdminManageDarCollections} props={props} rolesAllowed={[USER_ROLES.admin]} />
     <AuthenticatedRoute path="/dataset_catalog" component={DatasetCatalog} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.all]} />
```

Comment:
Not for this PR, but `[USER_ROLES.admin, USER_ROLES.all]` got me wondering about what does `all` mean and why is admin not included in `all`, if indeed it isn't.